src/Commands/MakeModule.php

```diff
@@ -113,5 +113,11 @@ public function handle()
         $this->line("Module {$this->moduleName} has been generated.");
         $this->line("  Add this to your config/app.php@providers");
         $this->info("  {$this->baseNamespace}\\{$this->moduleNamespace}\\Providers\\RouteServiceProvider::class");
+
+        Storage::extend('local', function($app, $config) {
```

Comment:
Move this on top, extending the storage will never work once Storage::disk() called